geometry/polyop.cpp

```diff
@@ -48,65 +49,6 @@ typedef bg::model::polygon<bgPoint, false, false> bgPolygon; // ccw, unclosed
 typedef bg::model::multi_polygon<bgPolygon> bgMultiPolygon;
```

Comment:
three of these typedefs no longer necessary?

Reply:
still necessary after latest commit

Reply:
still necessary after latest commit